### PR TITLE
adding super librarians and 2024 fellows

### PR DESCRIPTION
**Pull request by mekarpeles**
Updates json for team page re: #9352

### CODE REVIEW

File: openlibrary/templates/about/team.json [hunk 1]
```diff
@@ -345,11 +496,34 @@
...
+  },
+  {
+    "name": "Stef K.",
+    "title": "Engineering Fellow",
+    "roles": ["fellow-2024"],
+    "departments": ["engineering"],
+    "projects": ["Wikisource Imports"],
+    "photo_path": "",
+    "github_id": "pidgezero-one",
+    "ol_key": ""
```
Comment (mekarpeles) [inline openlibrary/templates/about/team.json:519]:
```suggestion
    "name": "Stef Kischak",
    "title": "Engineering Fellow",
    "roles": ["fellow-2024"],
    "departments": ["engineering"],
    "projects": ["Wikisource Imports"],
    "photo_path": "https://github.com/user-attachments/assets/4d8dd2c9-7750-41f6-91e9-2ab5b27258cb",
    "personal_url": "https://pidgezero.one",
    "github_id": "pidgezero-one",
    "ol_key": ""
```

File: openlibrary/templates/about/team.json [hunk 2]
```diff
@@ -111,6 +133,26 @@
...
+  },
+  {
+    "name": "Freso",
+    "title": "Super Librarian",
+    "roles": ["volunteer"],
+    "departments": ["librarians"],
+    "projects": [],
+    "photo_path": "",
+    "github_id": "",
+    "ol_key": ""
```
Comment (Freso) [inline openlibrary/templates/about/team.json:154]:
```suggestion
    "name": "Frederik “Freso” S. Olesen",
    "title": "Super Librarian",
    "roles": ["volunteer"],
    "departments": ["librarians"],
    "projects": [],
    "photo_path": "https://avatars.githubusercontent.com/u/177659",
    "personal_url": "https://freso.dk/",
    "favorite_book_url": "https://openlibrary.org/people/freso/lists/OL272257L",
    "github_id": "Freso",
    "ol_key": "Freso"
```

**Review by Freso — CHANGES_REQUESTED** [review]
Hope this is okay! :bow:

File: openlibrary/templates/about/team.json [hunk 3]
```diff
@@ -63,12 +75,22 @@
...
   },
+  {
+    "name": "Daniel Capilla",
+    "title": "Super Librarian",
+    "roles": ["volunteer"],
+    "departments": ["librarians"],
+    "projects": [],
+    "photo_path": "",
+    "github_id": "",
+    "ol_key": ""
```
Comment (mekarpeles) [inline openlibrary/templates/about/team.json:86]:
```suggestion
  {
    "name": "Daniel Capilla",
    "title": "Super Librarian",
    "roles": ["volunteer"],
    "departments": ["librarians"],
    "projects": [],
    "photo_path": "https://avatars.githubusercontent.com/u/7125952",
    "github_id": "dcapillae",
    "personal_website": "https://dcapillae.wordpress.com",
    "ol_key": "dcapillae"
```

File: openlibrary/templates/about/team.json [hunk 4]
```diff
@@ -334,6 +479,16 @@
     "github_id": "szgrune",
     "ol_key": "samstupendous"
   },
+  {
+    "name": "Sandy C.",
+    "title": "Engineering Volunteer",
+    "roles": ["volunteer"],
+    "departments": ["engineering"],
+    "projects": ["Vue Migrations"],
+    "photo_path": "",
```
Comment (mekarpeles) [inline openlibrary/templates/about/team.json:488]:
```suggestion
    "photo_path": "https://github.com/user-attachments/assets/91b60336-05bf-48d7-9307-7654465e7434",
```

**Pull request merged**